frontend/src/components/ViewTrips/trip.js

```diff
@@ -1,4 +1,25 @@
 import React from 'react';
+import Button from 'react-bootstrap/Button';
+
+import ViewActivitiesButton from './view-activities-button.js';
+
+/**
+ * Returns the date range of the trip associated with the Trip document data
+ * `tripObj`.
+ *
+ * Note: tripObj will always contain valid start_date and end_date fields.
+ *
+ * @param {firebase.firestore.DocumentData} tripObj Object containing the fields
+ *    and values for a Trip document.
+ * @return Date range of the trip (if it exists).
+ */
+export function getDateRange(tripObj) {
+  const startDate = tripObj.start_date.toDate();
+  const endDate = tripObj.end_date.toDate();
+  return `${startDate.getMonth() + 1}/${startDate.getDate()}/`  +
+      `${startDate.getFullYear()} - ${endDate.getMonth() + 1}/` +
+      `${endDate.getDate()}/${endDate.getFullYear()}`;
+}
```

Comment:
Why is 1 added to the start and end dates?

Additionally, what happens if the starting month is 12? Does +1 make it 13?

Reply:
The months are zero indexed with js dates. So the +1 is added so the month displays the correct month number.

Reply:
I will make a note in the function description.